### DISCUSSION TITLE
Authentik can’t be reached after installation

### DISCUSSION THREAD

**Discussion by Citroene** [community-scripts/ProxmoxVE#1071] [Q&A] [answered]
Hi,

I installed Authentik with the new script on my homeserver. when the script is done,  i get this this url "http://X.X.X.X/if/flow/initial-setup/"

when i go to the url, then i get this error in chrome.

```
This site can’t be reached
X.X.X.X refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED
```
The ip adress is reachable via ping in cmd
![Schermafbeelding 2024-12-28 203953](https://github.com/user-attachments/assets/d04c6b24-d693-448c-b089-5655dd0009ff)
My other self-hosted services are all accessible.

am i doing something wrong?

kind regards

**Comment by knario** [comment, marked as answer]
You should put the port to be able to access. I noticed it when I installed the script

**Reply by Citroene** [reply]
Thx!
